extensions/amp-lightbox/1.0/component.js

```diff
@@ -218,15 +206,10 @@ function CloseButton({close, customCloseButton = <ScreenReaderCloseButton />}) {
  * We do not want this in the tab order since it is not really "visible"
  * and would be confusing to tab to if not using a screen reader.
  *
+ * @param {!LightboxDef.CloseButtonProps} props
  * @return {PreactDef.Renderable}
  */
-function ScreenReaderCloseButton() {
+function ScreenReaderCloseButton(props) {
   const classes = useStyles();
-  return (
-    <button
-      ariaLabel={DEFAULT_CLOSE_LABEL}
-      tabIndex={-1}
-      className={classes.closeButton}
-    />
-  );
+  return <button {...props} tabIndex={-1} className={classes.closeButton} />;
```

Comment:
Would you say `tabIndex` belongs to the default props or nah?

Reply:
Nah -- the value of `-1` is specific to it being accessible to screen reader but not keyboard `Tab`. Whatever element that is given instead of this should have its default `tabindex` value.